doc/classes/InputEventAction.xml

```diff
@@ -35,8 +36,10 @@
 	</methods>
 	<members>
 		<member name="action" type="String" setter="set_action" getter="get_action">
+			Action's name. Each action is assigned and retrived by this [String].
 		</member>
 		<member name="pressed" type="bool" setter="set_pressed" getter="is_pressed">
+			[code]true[/code] if the action is pressed. [code]false[/code] if the action is released.
```

Comment:
See Writing Guidelines - boolean members should begin "If true":

If [code]true[/code] the action's state is pressed.

Reply:
My fault, I had forgotten about this and recommended this sentence to Jerome. Yeah gotta fix it

Reply:
Ok right. For the "false" part, I remove ? Would be nice to clarify to have a things saying about releasing...